components/collection/CollectionInfo.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <div
+    class="is-flex is-justify-content-space-between mobile-flex-direction-column gap">
+    <div class="is-flex is-flex-direction-column is-flex-grow-1 max-width">
+      <div v-if="collectionInfo?.currentOwner" class="is-flex mb-2">
+        <div class="mr-2">{{ $t('activity.creator') }}</div>
+        <nuxt-link :to="`/${urlPrefix}/u/${address}`" class="has-text-link">
+          <IdentityIndex ref="identity" :address="address" />
+        </nuxt-link>
+      </div>
+      <div>
+        {{ collectionInfo?.meta.description }}
+      </div>
+    </div>
+    <div>
+      <div class="columns is-mobile">
+        <div class="column">
+          <CollectionInfoLine :title="$t('activity.network')" :value="chain" />
+          <CollectionInfoLine
+            title="Items"
+            :value="representation(stats.collectionLength)" />
+          <CollectionInfoLine
+            :title="$t('series.owners')"
+            :value="representation(stats.uniqueOwners)" />
+        </div>
+        <div class="column">
+          <CollectionInfoLine :title="$t('activity.floor')">
+            <CommonTokenMoney
+              :value="stats.collectionFloorPrice"
+              inline
+              :round="2" />
+          </CollectionInfoLine>
+          <CollectionInfoLine
+            v-if="stats.bestOffer"
+            :title="$t('activity.bestOffer')">
+            <CommonTokenMoney :value="stats.bestOffer" inline :round="2" />
+          </CollectionInfoLine>
+          <CollectionInfoLine :title="$t('activity.volume')">
+            <CommonTokenMoney
+              :value="stats.collectionTradedVolumeNumber"
+              inline
+              :round="2" />
+          </CollectionInfoLine>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+<script setup lang="ts">
+import CollectionInfoLine from './collectionInfoLine.vue'
+import CommonTokenMoney from '@/components/shared/CommonTokenMoney.vue'
+import IdentityIndex from '@/components/identity/IdentityIndex.vue'
+import {
+  useCollectionDetails,
+  useCollectionMinimal,
+} from './utils/useCollectionDetails'
+
+const route = useRoute()
+const { urlPrefix } = usePrefix()
+const { availableChains } = useChain()
+
+const collectionId = computed(() => route.params.id)
+const chain = computed(
+  () =>
+    availableChains.value.find((chain) => chain.value === route.params.prefix)
+      .text
+)
+const address = computed(() => collectionInfo.value?.currentOwner)
+
+const { collection: collectionInfo } = useCollectionMinimal({
+  collectionId: collectionId.value,
+})
+const { stats } = useCollectionDetails({ collectionId: collectionId.value })
+
+const representation = (value: number | undefined): string => {
+  if (value == undefined) {
+    return '0'
+  }
+
+  if (value >= 1000) {
+    return (value / 1000).toFixed(2) + 'K'
+  }
+  return `${value}`
+}
```

Comment:
What does this app do ?

Reply:
Visual representation of a number.
For example 5237 =>5.23K

Reply:
IMO we should show full numbers

Reply:
I followed the figma design.
Plus what about mobile?

  Why you prefer full number?

Reply:
> Why you prefer full number?

It is unified across the app